vasarhelyia/Smogler
# Add AlertviewController prompt for API Token.

=What MR does:=
- Adds AlertViewController with textfield to prompt user to add his generated token.
 
# Where the reviewer should start:=
- `AppDelegate+ShakeMotion.swift`
 
# How to test this change:=
- Run the app on real device and shake it, you should see nice AlertViewController with prompt for API Token. Insert the token and wait for UI to refresh.
 

# Additional info:
I added `Info.plist` files to .gitignore file to prevent conflicts with codesigning since Xcode isn't properly fixing this while creating git repo... Please configure your `.gitignore` file accordingly (Or do as you please, this is just a suggestion 👍 ) 


```
# Info.plist files for bundle identifiers...
Smogler\ WatchKit\ Extension/Info.plist
Smogler\ WatchKit\ App/Info.plist
Smogler/Resources/Info.plist

## Token for API
Smogler/Service/APIService+Token.swift
```
